explorer.schema.yaml

```diff
@@ -0,0 +1,973 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+title: Diffix Explorer Output Schema
+description: Schema for data produced by Diffix Explorer
+type: object
+properties:
+  id:
+    type: string
+    format: uuid
+  status:
+    "$ref": "#/definitions/exploration_status"
+  versionInfo:
+    "$ref": "#/definitions/version_info"
+  dataSource:
+    type: string
+  table:
+    type: string
+  columns:
+    type: array
+    items:
+      "$ref": "#/definitions/column_metrics"
+  sampleData:
+    type: array
+    items:
+      type: array
```

Comment:
There is no way we can type the contents of the array?

Reply:
The sample data could contain any data type. The only constraint is that the data types in the sample data should match the data types of the columns. I am not sure that it is possible to express that in JSON schema, but I will look into it.

Reply:
If it's not immediately obvious, then let's ignore it.